xesmf/tests/test_frontend.py

```diff
@@ -203,7 +238,7 @@ def test_build_regridder_from_dict():
 
 def test_regrid_periodic_wrong():
     # not using periodic option
-    regridder = xe.Regridder(ds_in, ds_out, 'bilinear')
+    regridder = xe.Regridder(ds_in, ds_out, 'bilinear', default_weights=False)
```

Comment:
same comment as above

Reply:
For this test I added `default_weights=False` as it demonstrates how the missing periodic option can lead to unmapped cells. The test is set up in a way that the unmapped cells need a value of 0 and not `np.nan`.